src/openms/source/FORMAT/FileTypes.cpp

```diff
@@ -113,6 +113,7 @@ namespace OpenMS
     TypeNameBinding(FileTypes::NOVOR, "novor", "Novor custom parameter file"),
     TypeNameBinding(FileTypes::XQUESTXML, "xquest.xml", "xquest.xml file"),
     TypeNameBinding(FileTypes::SPECXML, "spec.xml", "spec.xml file"),
+    TypeNameBinding(FileTypes::SQLITE, "sqlite", "sqlite database file"),
```

Comment:
would sqAMS be better?

Reply:
I agree. Otherwise you don't really know what is inside unless you inspect it.
Thats like using '.xml' for all mzXML and mzML files

Reply:
Ok!